data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml

```diff
@@ -4,9 +4,17 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
     <changeSet author="R3.Corda" id="crypto-creation-v1.0">
         <createTable tableName="crypto_wrapping_key">
+            <column name="id" type="VARCHAR(64)">
+                <constraints nullable="false"/>
+            </column>
             <column name="alias" type="VARCHAR(64)">
                 <constraints nullable="false"/>
             </column>
+            <!-- generation of the wrapping key, as a UUID if created by us or a user supplied value for root wrapping keys 
```

Comment:
Maybe?
```suggestion
            <!-- generation of the wrapping key, as a INT with the highest value one considered the latest
```

Reply:
well, going with INT does solve the problem of the exact type of a UUID, but it does expose us to various risks around collisions.

Reply:
> well, going with INT does solve the problem of the exact type of a UUID, but it does expose us to various risks around collisions.

I don't think there is a risk of collision here. We're controlling what is inserted and when.
And it would be nice to be able to see which one is newest, so an INT makes sense.